lib/impress.client.js

```diff
@@ -79,44 +79,41 @@ const extCases = {
   html: (client, application) => { // eslint-disable-line
     if (typeof(client.context.data) === 'string') {
       client.end(client.context.data);
-    } else client.processingPage();
+    } else {
+      client.processingPage();
+    }
   },
 
-  sse: (client, application) => {
-    if (application.sse) {
-      client.sseConnect();
-    } else client.error(510);
+  sse: (client, application) => { // eslint-disable-line
```

Comment:
I'll remove second parameter `application` from all `extCases` and will use `client.application`